mongo/integration/database_test.go

```diff
@@ -40,27 +41,27 @@ func TestDatabase(t *testing.T) {
 
 	mt.RunOpts("run command", noClientOpts, func(mt *mtest.T) {
 		mt.Run("decode raw", func(mt *mtest.T) {
-			res, err := mt.DB.RunCommand(mtest.Background, bson.D{{"ismaster", 1}}).DecodeBytes()
+			res, err := mt.DB.RunCommand(mtest.Background, bson.D{{internal.LegacyHello, 1}}).DecodeBytes()
 			assert.Nil(mt, err, "RunCommand error: %v", err)
 
 			ok, err := res.LookupErr("ok")
 			assert.Nil(mt, err, "ok field not found in result")
 			assert.Equal(mt, bson.TypeDouble, ok.Type, "expected ok type %v, got %v", bson.TypeDouble, ok.Type)
 			assert.Equal(mt, 1.0, ok.Double(), "expected ok value 1.0, got %v", ok.Double())
 
-			isMaster, err := res.LookupErr("ismaster")
-			assert.Nil(mt, err, "ismaster field not found in result")
-			assert.Equal(mt, bson.TypeBoolean, isMaster.Type, "expected isMaster type %v, got %v", bson.TypeBoolean, isMaster.Type)
-			assert.True(mt, isMaster.Boolean(), "expected isMaster value true, got false")
+			hello, err := res.LookupErr(internal.LegacyHelloLowercase)
+			assert.Nil(mt, err, "legacy hello response field not found in result")
+			assert.Equal(mt, bson.TypeBoolean, hello.Type, "expected hello type %v, got %v", bson.TypeBoolean, hello.Type)
+			assert.True(mt, hello.Boolean(), "expected hello value true, got false")
 		})
 		mt.Run("decode struct", func(mt *mtest.T) {
 			result := struct {
-				IsMaster bool    `bson:"ismaster"`
-				Ok       float64 `bson:"ok"`
+				Hello bool    `bson:"ismaster"`
```

Comment:
Let me know if you have any clever ways of obfuscating or removing this `ismaster` struct tag. Wasn't sure how to do it without altering the underlying test.

Reply:
Great idea; switched to `ping` and removed the `Hello` field entirely.